docs/core/tools/troubleshoot-usage-issues.md

```diff
@@ -141,7 +141,9 @@ dotnet tool install -g --version 1.1.0-pre <toolName>
 
 If you try to install a NuGet package that is a regular NuGet package and not a .NET Core tool, you'll see an error similar to the following:
 
-`NU1212: Invalid project-package combination for `<ToolName>`. DotnetToolReference project style can only contain references of the DotnetTool type.`
+```
```

Comment:
changing to this format makes the scrollbar appear. Probably better to keep with the backticks.

Reply:
instead of the code fence, would a blockquote make more sense?
Because of the internal "`<ToolName>`" an double or triple backtick will be needed, but could stay as a sincle line instead of a block

Reply:
Sure, you can change it to a blockquote.